Description: enhance default shutdown behavior
 Modify the default config to do 10 parallel shutdown requests and reduce
 the timeout to 120s/2m.
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.1/tools/libvirt-guests.sysconf
===================================================================
--- libvirt-1.3.1.orig/tools/libvirt-guests.sysconf	2016-03-01 10:52:53.000000000 +0100
+++ libvirt-1.3.1/tools/libvirt-guests.sysconf	2016-03-04 14:44:35.802622104 +0100
@@ -26,14 +26,14 @@
 
 # If set to non-zero, shutdown will suspend guests concurrently. Number of
 # guests on shutdown at any time will not exceed number set in this variable.
-#PARALLEL_SHUTDOWN=0
+PARALLEL_SHUTDOWN=10
 
 # Number of seconds we're willing to wait for a guest to shut down. If parallel
 # shutdown is enabled, this timeout applies as a timeout for shutting down all
 # guests on a single URI defined in the variable URIS. If this is 0, then there
 # is no time out (use with caution, as guests might not respond to a shutdown
 # request). The default value is 300 seconds (5 minutes).
-#SHUTDOWN_TIMEOUT=300
+SHUTDOWN_TIMEOUT=120
 
 # If non-zero, try to bypass the file system cache when saving and
 # restoring guests, even though this may give slower operation for
